inc/uf2.h

```diff
@@ -133,10 +142,10 @@
 #define COLOR_UART 0x040400
 #define COLOR_LEAVE 0x000000
 #else
-#define COLOR_START 0x000040
-#define COLOR_USB 0x004000
-#define COLOR_UART 0x404000
-#define COLOR_LEAVE 0x400040
+#define COLOR_START 0x00000A
```

Comment:
What made you change the color intensities? These were chosen to accommodate different intensities of APA102 and WS2812 RGB leds, including differences in production runs.